src/controllers/main/main.ts

```diff
@@ -1345,6 +1348,20 @@ export class MainController extends EventEmitter {
     this.emitUpdate()
   }
 
+  // a method in the main controller to understand whether
+  // signAccountOp is still initialized
+  isSignRequestStillActive(op: AccountOp): boolean {
+    const accountOpIndex = this.actions.actionsQueue.findIndex(
+      (a) => a.type === 'accountOp' && a.id === `${op.accountAddr}-${op.networkId}`
```

Comment:
this function will work only for SA accountOps. The BA actions of type 'calls' have the same id as their userRequest which in most cases is a number